employees/views.py

```diff
@@ -192,13 +194,50 @@ def get_queryset(self) -> QuerySet:
         )
 
     def get_context_data(self, **kwargs: Any) -> dict:
+        def get_expected_hours_sum_for_all_projects(projects: QuerySet) -> Tuple[dict, datetime.timedelta]:
```

Comment:
Please don't next methods inside other methods. Please do them next to other methods.